sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemClient.java

```diff
@@ -129,7 +124,7 @@ public DataLakeDirectoryClient getDirectoryClient(String directoryName) {
      *
      * @return A new {@link DataLakeDirectoryClient} object which references the root directory in this file system.
      */
-    public DataLakeDirectoryClient getRootDirectoryClient() {
```

Comment:
Do you have tests about getRootDirectoryClient, do you need to edit them?

Reply:
Our tests are in Groovy and it lets you test private methods too - so it shouldnt break them!